<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CropUseDialogClass</class>
 <widget class="QDialog" name="CropUseDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Crop Use</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/pic/crop16.png</normaloff>:/pic/crop16.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxCrop"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Crop Use Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditCropUse"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Harvest Unit:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboUnit">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string notr="true">dt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">t</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">kg</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Withdrawal kg/unit (use positve figures e.g. in case of N fixation)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string notr="true">N:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="dsbN">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string notr="true">P2O5:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="dsbP2O5">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string notr="true">K2O:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="dsbK2O">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string notr="true">MgO:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="dsbMgO">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string notr="true">S:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="dsbS">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string notr="true">CaO:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QDoubleSpinBox" name="dsbCaO">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Humus:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDoubleSpinBox" name="dsbHumus">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>&amp;Crop:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CropUseDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CropUseDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxCrop</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
